gef-ui/server/controllers/supporting-information/manual-inclusion-questionnaire/index.js

```diff
@@ -0,0 +1,237 @@
+const Application = require('../../../models/application');
+const { validationErrorHandler } = require('../../../utils/helpers');
+const validateFile = require('../../../utils/validateFile');
+const { uploadAndSaveToDeal, removeFileFromDeal } = require('../../../utils/fileUtils');
+
+// This is only needed in this question due to the different size requirement
+const MAX_FILE_SIZE = 1024 * 1024 * 12;
+const FIELD_NAME = 'manualInclusion';
+
+const getUploadManualInclusion = async (req, res) => {
+  const {
+    session: { user, userToken },
+    params: { applicationId },
+  } = req;
+
+  const application = await Application.findById(applicationId, user, userToken);
+
+  if (!application) {
+    console.log(`User unauthorised to view application ${applicationId} manual inclusion`);
```

Comment:
is this static msg always valid when we dont have application returned?

Reply:
It's what we've used elsewhere so I followed that pattern

Reply:
fair enough :+1: